teuthology/repo_utils.py

```diff
@@ -440,6 +438,9 @@ def fetch_teuthology(branch, commit=None, lock=True):
 
 
 def bootstrap_teuthology(dest_path):
+    # avoid circular imports
+    import teuthology.exporter as exporter
+
     with exporter.BootstrapTime.time():
```

Comment:
I'm just thinking, why we use anything from exporter in repo_utils.

Reply:
This is what enables the exporter to report the time it takes for bootstrap to complete